pkgs/development/python-modules/flufl/lock.nix

```diff
@@ -1,13 +1,22 @@
-{ buildPythonPackage, fetchPypi, atpublic }:
+{ lib, buildPythonPackage, fetchPypi, atpublic, psutil, pytest }:
 
 buildPythonPackage rec {
   pname = "flufl.lock";
-  version = "3.2";
-
-  propagatedBuildInputs = [ atpublic ];
+  version = "5.0.5";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0nzzd6l30ff6cwsrlrb94xzfja4wkyrqv3ydc6cz0hdbr766mmm8";
+    sha256 = "1bnapkg99r6mixn3kh314bqcfk8q54y0cvhjpj87j7dhjpsakfpz";
+  };
+
+  propagatedBuildInputs = [ atpublic psutil ];
+  checkInputs = [ pytest ];
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

Reply:
The manual says

> pytestCheckHook is a convenient hook which will substitute the setuptools test command for a checkPhase which runs pytest. This is also beneficial when a package may need many items disabled to run the test suite.

Why should I use it here?

Reply:
Because setuptools test will no longer be called in some future.

Reply:
Thanks for explaining.  Could we get that in the manual maybe?